src/mongo/db/pipeline/expression.cpp

```diff
@@ -26,7 +26,6 @@
  * it in the license file.
  */
 
-
```

Comment:
Please avoid stray whitespace changes. It makes it harder to review and complicates the git blame unnecessarily. Not generally a big deal, but please avoid it going forward.